src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/CommonCryptographyManager.cs

```diff
@@ -2,20 +2,28 @@
 // Licensed under the MIT License.
 
 using System;
+using System.Collections.Concurrent;
 using System.Globalization;
 using System.Security.Cryptography;
 using System.Security.Cryptography.X509Certificates;
 using System.Text;
 using Microsoft.Identity.Client.Internal;
 using Microsoft.Identity.Client.Internal.ClientCredential;
 using Microsoft.Identity.Client.PlatformsCommon.Interfaces;
-using Microsoft.Identity.Client.PlatformsCommon.Shared;
 using Microsoft.Identity.Client.Utils;
 
-namespace Microsoft.Identity.Client.Platforms.netcore
+namespace Microsoft.Identity.Client.PlatformsCommon.Shared
 {
-    internal class NetCoreCryptographyManager : ICryptographyManager
+
+#if ANDROID && !MAUI
+    [global::Android.Runtime.Preserve(AllMembers = true)]
+#endif
+    [Preserve(AllMembers = true)]
+    internal class CommonCryptographyManager : ICryptographyManager
```

Comment:
Why were the crypto managers refactored?

Reply:
I did that, sorry, I couldn't help it :). There is nothing differet in Android, ios, mac etc. crypto managers.